src/PrestaShopBundle/Resources/views/Admin/Module/Includes/action_menu.html.twig

```diff
@@ -1,25 +1,25 @@
 <div class="pull-right btn-group">
 
-    {% if module.attributes.get('url_active') == 'buy' %}
-        <a class="btn btn-primary btn-sm light-button module_action_menu_go_to_addons" href="{{module.attributes.get('url') }}" target="_blank">
-            {% if module.attributes.has('price') and module.attributes.get('price').EUR > 0 %}{{module.attributes.get('price').EUR}}{% else %}Discover{% endif %}
+    {% if module.attributes.url_active == 'buy' %}
+        <a class="btn btn-primary btn-sm light-button module_action_menu_go_to_addons" href="{{module.attributes.url }}" target="_blank">
+            {% if module.attributes.price.raw != '0.00' %}{{module.attributes.price.displayPrice}}{% else %}Discover{% endif %}
```

Comment:
who have introduced this option ? the main idea of presenter is to avoid any calculation operations on templates :disappointed: